api/scripts/certification/target-profile-to-calibrated-framework.js

```diff
@@ -0,0 +1,46 @@
+import { knex } from '../../db/knex-database-connection.js';
+import { usecases } from '../../src/certification/configuration/domain/usecases/index.js';
+import { ComplementaryCertificationKeys } from '../../src/certification/shared/domain/models/ComplementaryCertificationKeys.js';
+import { Script } from '../../src/shared/application/scripts/script.js';
+import { ScriptRunner } from '../../src/shared/application/scripts/script-runner.js';
+
+export class TargetProfileToCalibratedFrameworkScript extends Script {
+  constructor() {
+    super({
+      description: 'Use target profile to create a consolidated framework',
+      permanent: false,
+      options: {
+        complementaryCertificationKey: {
+          type: 'string',
+          describe: 'Certification key',
```

Comment:
Une certification m'evoque un resultat de certif pas une complementaire

```suggestion
          describe: 'Complementary Certification key',
```

Voir je mettrais

```suggestion
          describe: 'Consolidated framework identifier', // ou bien key
```